public/cli/lib/commands/convert.js

```diff
@@ -195,7 +196,20 @@ Command.run = function(options) {
  function prepareLogger( done, numLogsToKeep=10 ) {
 
     // make sure the logging directory exists
-    var pathToLogDir = path.join(__dirname, "..", "..", "..", "..", "logs");
+    var pathToLogDir;
+    switch(process.platform) {
+        case "darwin":
+            pathToLogDir = path.join(os.homedir(), "Library", "Logs", "bbk", "logs");
```

Comment:
Th MacOS doesn't need the final `logs` folder according to Eric's info.

Reply:
yeah, but for consistency sake, do you want to take ```"logs"``` out of MacOS, but leave it in for the rest?

Reply:
Yes, as it is already in a specific `Logs` folder. On Mac it is ordered by logs first then app; on Linux and Windows it is ordered by app first then logs.